packages/core/supabase-js/TESTING.md

```
@@ -66,194 +79,253 @@ npx nx test:coverage supabase-js
+
+## Platform-Specific Testing (CI Only)
+
+Platform tests verify that the SDK works correctly across different runtime environments (Node.js, Deno, Bun, Expo, Next.js). These tests run **automatically in CI** on every pull request using [pkg.pr.new](https://github.com/stackblitz-labs/pkg.pr.new) for package publishing.
+
+### How Platform Tests Work in CI
 
-```bash
-# This will run `npm i` in each of the intergration tests directories,
-# fetching the locally published packages
-npx nx update:test-deps supabase-js
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)